cli/asconfig.json

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
This is duplicating quite a lot of text that must be kept in sync at multiple places now. Or do you have something in mind already to avoid that?

Reply:
Since all of these match the descriptions, I suppose that we could just remove the descriptions here. I was hoping for more feedback when I asked for it, but I decided to try and implement something

Reply:
Yeah I should remove the descriptions. This was just a lot of copy + paste action from asc.json.